public/themes/default/pages/editor.html

```diff
@@ -69,8 +69,16 @@
 
         <file-tree
           id="filetree"
-          {% if not projectMember %}readonly{% endif %}
+          {% if not user.superuser and not projectMember %}
+            class="full-height" readonly
+          {% endif %}
         ></file-tree>
+
+        {% if user.superuser or projectMember %}
```

Comment:
honestly, this is used so much that it should just be its own `set` command with a single var, so we don't need the clumsy `if not user.superuser and not projectMember` but can just use `if editRights` / `if not editRights` or something.